Makefile

```diff
@@ -135,18 +135,21 @@ build-local-container: build ## Builds the provisioner container image using loc
 	$(CONTAINER_RUNTIME) build -f Dockerfile.local -t $(IMAGE) .
 
 kind-load-bundles:
+	-mkdir $(TESTDATA_DIR)/bundles/plain-v0/subdir/manifests/emptydir
```

Comment:
Since we already have `subdir/manifests/subdir` checked in, do we actually need `emptydir` to exercise the logic we expect? I'm curious if we can remove this line?

If we _do_ need the emptydir, I'd suggest checking it in with a `.gitignore` and a `.dockerignore` like we have with the [no-manifests](https://github.com/operator-framework/rukpak/tree/main/testdata/bundles/plain-v0/no-manifests) bundle.

Reply:
Thank! I thought that It's better checking the empty directory doesn't cause any issue.  I can take it out.  
I tried checking in the `emptydir` in the `.gitignore` but it actually caused failure.  The manifest reader reads it and errored.  I'm thinking about opening an issue for it.

Reply:
> The manifest reader reads it and errored. I'm thinking about opening an issue for it.

I might be misunderstanding the issue, but I don't think the .gitignore file should be showing up in the container that the unpacker sees if you exclude the .gitignore file from the docker build with a .dockerignore file.

Reply:
I missed the `.dockerignore` to remove the `.gitignore`.  I'll try.  Thanks!